doc/next/_sources/overview.rst.txt

```diff
@@ -238,10 +239,13 @@ The API partner can also fetch the ad or campaign _by_ `vendorId` - see :ref:`ge
 for details - to  get the necessary primary key to perform other operations through the API.
 This means the API partner does not need to keep an explicit mapping between their own primary key and the one generated
 by our system, making integration simpler.
+
 Such a vendorId cannot, once set, be changed or removed. It has become as strong as the primary key, in combination with the seller account
 that's being used. This also means that within all ads of a seller, the vendorIds must be unique. Similarly, within all campaigns
 of a seller, the vendorIds must be unique.
 
+Keep in mind that vendorId is case-insensitive and only allows `windows-1252`_ (also known as Latin-1) characters.
```

Comment:
Can we expand on the implications of the case-insensitivity? how is it surfaced in getter methods?

Reply:
we can add some example like:
For example, an ad is created with vendorId "abc123", and this ad can be retrieved by vendorId "Abc123". If you want to create another ad with vendorId "aBc123", the creation will fail. Because "abc123", "Abc123" and "aBc123" are identical.
what do you think?

Reply:
Yeah that sounds good!
I would also add a note on how the payload would look like on ad retrieval. If you fetch an ad (by vendor or id or some slicing criteria) is it always going to be surfaced in lowercase?

Reply:
The retrieval will have the same capitalization used on insertion. Example:
ad is created with vendorID `Abc123`, that ad can be retrieved using vendorID `abc123` (lower case A) but the payload will be returned with vendorID `Abc123`, same case as used on insertion.

Let me write something about this in the docs.